app/models/foreman_openscap/arf_report.rb

```diff
@@ -113,9 +113,13 @@ def self.create_arf(asset, params)
           params[:logs].each do |log|
             src = Source.find_or_create(log[:source])
             digest = Digest::SHA1.hexdigest(log[:title])
-            msg = Message.where(:value => N_(log[:title]), :digest => digest, :severity => log[:severity],
-                                :description => newline_to_space(log[:description]), :rationale => newline_to_space(log[:rationale]),
-                                :scap_references => references_links(log[:references])).first_or_create
+            msg = Message.where(:digest => digest).first_or_create do |message|
+              message.value = N_(log[:title])
+              message.severity = log[:severity]
+              message.description = newline_to_space(log[:description])
```

Comment:
does this mean that when newer version of the rule description is defined in openscap, we won't see the update here? should we perhaps update all attributes that changed for the given log title? I've been told that these things change.